Add Contact Us page & update footer links (#33)
web3-storage/docs: docs/community/contact-us.md

---

> title: Contact Us

Reviewer: ```suggestion
title: Help & support
```
for consistency with other properties, and for sentence case rather than title case

description: Get help with Web3.Storage
---

# Contact Us

> If you're having trouble with the Web3.Storage service, we're here to help.

Reviewer: ```suggestion
If you're having trouble with the Web3.Storage service, or just want to get help and support from the global Web3 developer community, here's where to start.
```

> **Have a question?**

Reviewer: ```suggestion
## Questions and community discussion
```

> Join us in the **#web3-storage** channel on [Slack!](https://filecoin.io/slack)

Reviewer: ```suggestion
Have a question? Want to discuss problems, opportunities, or ideas? Join your peers in the Web3 developer community in the **#web3-storage** channel on [Slack.](https://filecoin.io/slack)
```

Author: @jessicaschilling I'm in support of adding additional context here, but I didn't think the channel was for discussing Web3 general stuff, but rather for discussing the Web3.Storage service and how to use it. @jnthnvctr what's your take?

Reviewer: @terichadbourne Maybe just this then?
`Have a question about using Web3.Storage? Want to discuss problems, opportunities, or ideas for integrating Web3.Storage into your project or product? Join your peers in the developer community in the **#web3-storage** channel on [Slack.](https://filecoin.io/slack)`

Author: Love it!

> **Found a bug or have a feature request?**

Reviewer: ```suggestion
## Bug reports or feature requests
```

> [File an issue!](https://github.com/web3-storage/web3.storage/issues/new/choose)

Reviewer: ```suggestion
Found a bug or have a feature request? File an issue and make your voice heard.
- [File an issue for Web3.Storage docs](https://github.com/web3-storage/docs/issues/new/choose)
- [File an issue for Web3.Storage itself](https://github.com/web3-storage/web3.storage/issues/new/choose)
```

Author: @jessicaschilling Would you be opposed to swapping the order to put service issues above docs issues? In which case we'd describe it as "the Web3.Storage service"?

Reviewer: @terichadbourne sure, that makes sense!

> **Need more storage?**

Reviewer: ```suggestion
## Request more storage space
```

> [Shoot us a note.](mailto:web3storage@protocol.ai)

Reviewer: ```suggestion
Need more storage? [Shoot us a note.](mailto:web3storage@protocol.ai)
```
